syntax = "proto3";

package iotbridge;

message TemperatureRequest {
	string room_id = 1;
	float temperature = 2;
	string status = 3;
}

message TemperatureResponse {
	bool success = 1;
	string message = 2;
}

service SensorService {
	rpc SendTemperature(TemperatureRequest) returns (TemperatureResponse);
}
